README.md

```diff
@@ -63,30 +67,14 @@ The minimum amount of voting power require to pass a proposal of each type it's
 
 ### database setup
 
-check if you have postgres installed
+Make sure you have postgres running.
 
+To create the db, run in the terminal
 ```bash
-brew list
-```
-
-if you don't, install it (don't forget to `brew update`)
-
-```bash
-brew install postgres
-```
-
-start postgresql
-
-```bash
-brew services start postgresql
-```
-
-you can run this to check if everything is ok 
-```bash
-createdb `whoami` 
-``` 
+createdb -U >USER< snapshot
+````
 
-inside psql run `create database snapshot`
+the `default user` is postgres and the `default password` is postgres as well
```

Comment:
Missing period at the end of sentence. Also I'd capitalize first letter. (Same in line 79 if you want to change it)